### PR TITLE
Add import test script

### PR DESCRIPTION
**Pull request by spectranaut**
Make sure your db is up to date and run `yarn` to get new packages! Then run via: `yarn workspace server import-test`

### CODE REVIEW

File: server/scripts/import-tests/index.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+        let exampleDirs = fse.readdirSync(testDirectory);
+        for (let i = 0; i < exampleDirs.length; i++) {
+            const exampleDir = exampleDirs[i];
+            const subDirFullPath = path.join(testDirectory, exampleDir);
+            const stat = fse.statSync(subDirFullPath);
+
+            if (stat.isDirectory() && exampleDir !== 'resources') {
+                const exampleID = await this.upsertAPGExample(
+                    exampleDir,
+                    exampleName[exampleDir],
```
Comment (evmiguel) [inline server/scripts/import-tests/index.js]: In the `tests/` directory in https://github.com/w3c/aria-at/tree/master/tests, there are more examples in the directory than in `support.json`. When `exampleName[exampleDir]` doesn't exist, that data will be null in the table. Is that ok?

File: server/scripts/import-tests/index.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+  No arguments:
+    Will fetch most recent aria-at tests to update database.
+  Arguments:
+    -h, --help
+       Show this message.
+`);
+    process.exit();
+}
+
+const client = new Client();
```
[the remark this answers is not in the document]
Comment (spectranaut) [reply server/scripts/import-tests/index.js:27]: bash exports! I guess I should export those vars in the package.json

File: server/scripts/import-tests/index.js (hunk 1 above)
Comment (spectranaut) [reply server/scripts/import-tests/index.js:82]: yup that right, good catch. We are importing all tests whether or not there is a entry in support.json. The application will fall back to showing the directory name if it doesn't have a fancy name for the APG example from the support.json file. I figured that admins configuring test runs might want to see all tests regardless of what is in support.json.

File: server/scripts/import-tests/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (spectranaut) [reply server/scripts/import-tests/index.js:27]: I tried this: 
```
```

[the remark this answers is not in the document]
Comment (evmiguel) [reply server/scripts/import-tests/index.js:27]: @spectranaut `export $(cat config/dev.env); yarn workspace server import-tests`

**Pull request merged**